### DISCUSSION TITLE
How do I land a job after working through the biostars book?

### DISCUSSION THREAD

**Discussion by dgodin19** [ialbert/biostar-central#949] [General]
I'm working through biostars and really enjoying it. It has really helped me brush up on old skills and learn new ones. 

However, I would like to translate this into knowledge that I can show to an employer. Would the way one go about this is by trying your own data analysis and then upload it to github?

**Comment by ialbert** [comment]
I agree that a repository where you publish one or more reanalyses of existing scientific works or even produce new results would be a compelling way to demonstrate that you understand the subject.

For me code and processes that a candidate publishes on GitHub always feel more meaningful than recommendations letters or other items on a CV.

**Comment by tapanrathod** [comment]
Yes, absolutely. Transforming book exercises into independent data analyses on GitHub is one of the best ways to prove your skills to employers. Instead of just mirroring textbook code, choose real-world public datasets—such as those from NCBI GEO or Ensembl—and run your own pipelines or attempt to reproduce findings from recent literature. To stand out, ensure your repositories feature clear documentation with a thorough README.md, clean code structure, and reproducible environments using container or environment files.

​Beyond GitHub, bridging the gap between technical execution and career opportunities comes down to visibility. Share high-level summaries and data visualizations from your projects on LinkedIn or bioinformatics forums, and link your top repositories directly on your resume. Showing hiring managers that you can formulate an analytical question, execute a workflow independently, and clearly communicate your findings holds far more weight than standard course certificates.
